multivac/gather_job_data.py

```diff
@@ -162,6 +164,53 @@ def gather_data(self):
                 )
             self.gathered_data[job_id] = gathered_job_data
 
+    def put_to_db(self):
+        bucket = os.getenv('INFLUX_BUCKET')
+        org = os.getenv('INFLUX_ORG')
+
+        write_api = influx_connector()
+
+        for job in list(self.gathered_data.keys()):
+            curr_job = self.gathered_data[job]
+
+            time_to_datetime = datetime.fromisoformat(
+                f"{curr_job['queued_at'].rstrip('Z')}+00:00")
```

Comment:
Is this a conversion between two data formats?
Could you please specify the name of the target format at least?

Reply:
It's just convert string to datetime objet, not to special format. I'll make a comment

Reply:
thanks!